rpc/src/v1/impls/chain.rs

```diff
@@ -189,6 +189,17 @@ where
         Ok(self.client.balance(address, block_id.into()).map(Into::into))
     }
 
+    fn get_error_hint(&self, transaction_hash: H256) -> Result<Option<String>> {
+        let invoice_op = self.client.parcel_invoice(&transaction_hash.into());
```

Comment:
This can be simplified as below.
```rust
        if let Some(Invoice::Failure(error_string)) = self.client.parcel_invoice(&transaction_hash.into()) {             
            Ok(Some(error_string))                                                                                       
        } else {                                                                                                         
            Ok(None)                                                                                                     
        }                                                                                                                
```

Reply:
Simplified.